agent/multi_agent.py

```
@@ -8,155 +8,196 @@
+                    output_str = json.dumps(output) if isinstance(output, (dict, list)) else str(output)
+                    tool_outputs.append({"tool_call_id": tool_call.id, "output": output_str})
+
+                except Exception as e:
+                    print(f"Error calling tool {function_name}: {e}")
+                    tool_outputs.append({"tool_call_id": tool_call.id, "output": f"Error: {e}"})
+            else:
+                tool_outputs.append({"tool_call_id": tool_call.id, "output": f"Error: Tool '{function_name}' not found."})
+        
+        return tool_outputs
+
+    def _create_assistant(self, name: str, instructions: str, model: str = "gpt-4o", tools: list = [], response_format: Optional[dict] = None):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)